### PR TITLE
[Term Entry] Kotlin String Function: .drop()

### PR DESCRIPTION
**Pull request by MarkDGraham**
### Description

- Added the String.drop() term for Kotlin.
- Closes issue #3722 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Review by ishg-153 — CHANGES_REQUESTED** [review]
@MarkDGraham This looks great!
You do need to remove the `yarn.lock` file. If you're having trouble with that, let me know.

**Comment by MarkDGraham** [comment]
I tried to remove the yarn.lock using git rm but now a new branch is existing causing my branch to be ahead by 4 commits. Can you help @ishg-153?

**Comment by ishg-153** [comment]
> I tried to remove the yarn.lock using git rm but now a new branch is existing causing my branch to be ahead by 4 commits. Can you help @ishg-153?

I see. The best way would be to revert back to the commit before you had tried to remove the `yarn.lock` file. Then you will need to `git rm --cached yarn.lock`. This will make sure that the `yarn.lock` file is removed from the git repository but stays in your file system.

If you are still not able to do it yourself, I can take care of it for you. No need to fret.

**Comment by MarkDGraham** [comment]
@ishg-153 yarn.lock has been removed and should be good to go.

**Review by ishg-153 — APPROVED** [review]
This looks good for a second review.

File: content/kotlin/concepts/strings/terms/drop/drop.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+Title: '.drop()'
+Description: 'Returns a list containing all elements excluding 0th to nth elements.'
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/strings/terms/drop/drop.md:3]:
```suggestion
Description: 'Returns a string where the characters have been removed from the start to the nth index given.'
```

File: content/kotlin/concepts/strings/terms/drop/drop.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+Tags:
+  - 'Kotlin'
+  - 'Strings'
+  - 'Function'
+CatalogContent:
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+The **`.drop()`** method removes all elements from the original or 0th index element to the nth index element.
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/strings/terms/drop/drop.md:16]:
```suggestion
The **`.drop()`** method removes characters from the start (0th index) of a given string to the nth index element.
```

File: content/kotlin/concepts/strings/terms/drop/drop.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+## Syntax
+
+```pseudo
+String.drop(nth-element)
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/strings/terms/drop/drop.md:21]:
```suggestion
String.drop(nth-index)
```

File: content/kotlin/concepts/strings/terms/drop/drop.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+```
+
+- `String`: The sequence of chars to be removed from and replaced.
+- `nth-element`: The index of the fartherest element to be removed from the front.
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/strings/terms/drop/drop.md:25]:
```suggestion
- `nth-index`: The method will remove characters up to this value.
```

File: content/kotlin/concepts/strings/terms/drop/drop.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+The **`.drop()`** method removes all elements from the original or 0th index element to the nth index element.
+
+## Syntax
+
+```pseudo
+String.drop(nth-element)
+```
+
+- `String`: The sequence of chars to be removed from and replaced.
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/strings/terms/drop/drop.md:24]:
```suggestion
- `String`: The string to be modified. The method will return a new string with the specified values missing.
```

File: content/kotlin/concepts/strings/terms/drop/drop.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+- `nth-element`: The index of the fartherest element to be removed from the front.
+
+## Example
+
+The example demonstrates the use of `.drop()` to remove elements from the beginning of the string.
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/strings/terms/drop/drop.md:29]:
```suggestion
The example demonstrates the use of `.drop()` to remove characters from the beginning of the string.
```

File: content/kotlin/concepts/strings/terms/drop/drop.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+```kotlin
+fun main() {
+  val str = "This is new"
+  val str2 = "I'm a string"
+
+  str.drop(8) // drops 'This is '
+  str2.drop(6) // drops 'I'm a '
+
+  println(str)
+  println(str2)
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/strings/terms/drop/drop.md:40]:
```suggestion
  println(str.drop(8))   // Drops 'This is '
  println(str2.drop(6)) // Drops 'I'm a '
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@MarkDGraham I've added my comments. Please update the entry, and tag me when the changes are complete. Thanks

**Comment by MarkDGraham** [comment]
@caupolicandiaz I have made the changes following your comments and addressed them accordingly. I believe it should be ready now.

**Pull request merged**